doc/sfsmount.1.adoc

```diff
@@ -213,6 +216,10 @@ Equivalent to *--nostdopts* (*-n*) option for use in fstab.
 *-o nonempty*::
 Equivalent to *--nonempty* option for use in fstab.
 
+*-o sfsuseoldwritealgorithm*::
+Use legacy write algorithm. Though new algorithm is faster in most cases, old
+algorithm may prove useful in some scenarios.
```

Comment:
Consider mentioning the default value here.

Reply:
Applied.